app/models/draft-registration.ts

```diff
@@ -23,6 +32,15 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @belongsTo('registration-schema', { inverse: null })
     registrationSchema!: DS.PromiseObject<RegistrationSchemaModel> & RegistrationSchemaModel;
+
+    @hasMany('institution', { inverse: null })
+    affiliatedInstitutions!: DS.PromiseManyArray<InstitutionModel>;
+
+    @hasMany('subject', { inverse: null, async: false })
```

Comment:
Any particular reason why subject is `{async: false}` while other newly added relationships aren't? Should we make all new relationships synchronous?

Reply:
I vote yes.

Reply:
i assume it was copied from `./models/node`.

i have mixed feelings about using `async: false` for relationships that aren't auto-embedded -- lying to software often ends poorly. we started using sync relations for some of the reasons [in this fine article](https://embermap.com/notes/83-the-case-against-async-relationships), but without an intermediate layer (like ember-data-storefront) that we use consistently, i worry migrating to `async: false` might cause more trouble than it solves.

in this case, i was fine with `async: false` on `subjects` because all existing subjects code assumes it, but the code handling other relations (institutions, license) might assume the opposite -- we should absolutely aim for a more unified solution, but in this PR i'm fine leaving the status quo

Reply:
That's a good point about existing components that may not handle `async: false`. Most of the components we've created recently (and even not-so-recently), though, do their own explicit data loading. I think we should try it with `async: false` on all these relationships, and if there's problems, make a decision whether to make them async or update the component to handle `async: false`.